app/controllers/tool_groups_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ToolGroupsController < ApplicationController
+  before_action :authorize!
+
+  def index
+    render json: ToolGroup.all.order(name: :asc), status: :ok
```

Comment:
One of the aspects of jsonapi that we leverage is using the `include` parameter to dynamically including related objects we care about. In order for it to function we need to pass the parameter into the render logic.

```suggestion
    render json: ToolGroup.all.order(name: :asc), include: params[:include], status: :ok
```

Reply:
example from another controller: https://github.com/CruGlobal/mobile-content-api/blob/master/app/controllers/languages_controller.rb#L11

we should also include the fields param to be thorough.

Reply:
the include parameter only needs to be supported for the `index` and `show` methods